examples/job/get-file-from-job.ipynb

```diff
@@ -1,17 +1,19 @@
 {
```

Comment:
<p>There are still some references to "Exabyte ...", etc. I think we should adjust those to "Mat3ra ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>